### ISSUE TITLE
Change position of ‘Logout’ at the bottom after ‘Change Password’

### ISSUE THREAD

**Issue by keshakaneria** [anitab-org/vms#1031]
## Description
Position of `Logout` should be at the bottom after `Change password` button

## Mocks
![image](https://user-images.githubusercontent.com/46588494/91016847-0a4bf400-e60b-11ea-8957-06d2464b8a96.png)

### Update [Required]
- [ ] Logout Position changed

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
30 minutes

**Comment by priyanshurohilla** [comment]
can i work on this?

**Comment by keshakaneria** [comment]
Yes you can but i request you to wait until 1st September as this issue is under `Open Source Hack` tag. PRs for this tag will be evaluated after the event gets started. Kindly wait until then and will assign you after it @priyanshurohilla

**Comment by Devanshi-123** [comment]
Can more than one contributor work on an issue i also wish to work on this..

**Comment by arjundubey-cr** [comment]
@keshakaneria can anyone work on issues on only the assigned ones'?

**Comment by satya7289** [comment]
> @keshakaneria can anyone work on issues on only the assigned ones'?

Assigned ones'. Please wait until you get assigned by someone.

**Comment by priyanshurohilla** [comment]
> Yes you can but i request you to wait until 1st September as this issue is under `Open Source Hack` tag. PRs for this tag will be evaluated after the event gets started. Kindly wait until then and will assign you after it @priyanshurohilla

No problem i am participating in `open source hack`😁

**Comment by keshakaneria** [comment]
> Can more than one contributor work on an issue i also wish to work on this..

The assignee will be allowed to continue to work for three days. if he/she is not able to perform, then next interested assignee will be given this issue. At a time one assignee!

**Comment by yash2189** [comment]
can I work on this issue in OSH @keshakaneria

**Comment by keshakaneria** [comment]
You can surely work @yash2189 , if assigned person is you according to the first come first serve basis.

**Comment by yash2189** [comment]
can I work on this issue @keshakaneria?

**Comment by Abhijit5463** [comment]
Hey I am new in this community and I am participating in OSH can I take this up?

**Comment by Abhijit5463** [comment]
Isn't this violation of FCFS rule @keshakaneria ??

**Comment by yash2189** [comment]
@keshakaneria  as per FCFS rule  I had commented on the issue on 1st September. Isn't this a violation?
Please have a look on this.

**Comment by keshakaneria** [comment]
The person who is assigned was already working on this so I will have to fare with them too and have also raised an issue! Otherwise the FCFS rule will perfectly fine! Don't worry and keep working on other issues to work on :)

**Comment by ankitapuri** [comment]
Is this issue still open? If yes Please assign it to me @keshakaneria

**Comment by keshakaneria** [comment]
Yes one PR has been opened for this issue. You are welcomed to contribute to some other issues where no one is assigned :)

**Comment by keshakaneria** [comment]
> Is this issue still open? If yes Please assign it to me @keshakaneria

Do you want to take up this issue?

**Comment by ankitapuri** [comment]
Yes, I can try working on it @keshakaneria

**Comment by shubham582000** [comment]
can i work on this issue?
